packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/index.tsx

```diff
@@ -72,7 +71,7 @@ type PipelineHeaderProps = {
   onToggleOptions: () => void;
   isOpenPipelineVisible: boolean;
   isSavedPipelineVisible: boolean;
-  savedPipelines: Pipeline[];
+  savedPipelines: { id: string; name: string }[];
```

Comment:
Here and in a few other places: `Pipeline` from `modules/pipeline` is stage state, not saved pipeline type